client/my-sites/plan-features/index.jsx

```diff
@@ -197,6 +198,12 @@ class PlanFeatures extends Component {
 				'is-placeholder': isPlaceholder
 			} );
 
+			let description = planConstantObj.getDescription();
+
+			if ( abtest( 'plansWording' ) === 'targetedWording' ) {
+				description = planConstantObj.getTargetedDescription();
```

Comment:
We have a special function for adding abtests and any other modifications to the plans constants in `client/lib/plans` called `applyTestFiltersToPlansList`. All plans data go through this fn before they get to the `plan-features` component.

In [4b3420aabf4ee78faddd2d932e7b0a160a9c80f1](https://github.com/Automattic/wp-calypso/pull/7091/commits/4b3420aabf4ee78faddd2d932e7b0a160a9c80f1), I've moved this abtest to that fn and also fixed the issue with Jetpack plans. Could you please verify that it's working as you wanted?